lib/config-loader.js

```diff
@@ -72,6 +72,9 @@ function loadNamed(rootDir, env, name, mergeFn) {
     debug('found %s %s files', env, name);
     files.forEach(function(f) { debug('  %s', f); });
   }
+  if (name == 'config' && files.length === 0) {
+    console.warn('No config file was selected!');
```

Comment:
Please extend the warning with more information - what is the exact file name that was not found? What are the consequences for the user?

I think a better solution would be:
 - report this warning from the code which is looking up files
 - report the warning for all config files
 - report the warning only when the main `json` file is missing and there is a `local` or `{env}` file present

That should minimise the likelihood of reporting a false warning.